TLM/TLM/Compatibility/Check/DLCs.cs

```diff
@@ -42,7 +42,9 @@ public class DLCs {
                 }
 
                 Log.Info(sb.ToString());
-            } catch { }
+            }
+            catch {
```

Comment:
Maybe log something. Catch-alls are not nice, they hide real problems.

Reply:
Yup, will do.